rbc/src/main/java/com/library/rbc/model/Rental.java

```diff
@@ -0,0 +1,29 @@
+package com.library.rbc.model;
+
+import com.library.rbc.model.enums.RentalStatus;
+import com.library.rbc.model.enums.RentalType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "rentals")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Rental {
+
+    @Id
+    private String id;
+    private RentalType rentalType;
+    private String userId;
+    private Book book;
```

Comment:
When duplicating and embedding a subset of fields from the `Book` entity into the `Rental` entity, you should create a separate class for representing the embedded data, e.g. `BookMetadata`, and only include fields that need to be duplicated.